src/actions_notes.py

```diff
@@ -166,14 +171,12 @@ def add_note(args: list[str], notes_book: "NoteBook") -> None:
     return: str: Result message.
     """
 
-    name = args[0]
-    note = " ".join(args[1 : len(args)])
+    note_title: str = args[0]
+    note_body = " ".join(args[1:])
 
-    note_title: str = f"note-{len(notes_book.values()) + 1}"
-    notes_book.add(note_title, note)
-    notes_book.attach_to_contact(note_title, name)
+    new_note = notes_book.add(note_title, note_body)
+    note_table(new_note)
```

Comment:
do we need to print a table with only 1 added node
To not overload output I guess it will be better just to print Note with title blbla was created/replaced/deleted